client/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { ApolloProvider } from 'react-apollo';
+import ApolloClient from 'apollo-boost';
+import './App.css';
+import Property from './components/Property'
+const client = new ApolloClient({
+    uri : 'http://localhost:4000/graphql'
+});
+class App extends Component {
+
+ 
+  render() {
+    return (
+      <ApolloProvider client={client}>
+
+         <div className="App">
+         
+            <Property></Property>
```

Comment:
Since your data is a list of properties. The parent component should be named in plural form (`Properties`) and the child component where you give the individual property data should be named in singular form (`Property`).

```js
function Properties () {
  const properties = [
    { id: 1, name: 'My Property' },
  ];

  return properties.map(property => <Property property={property} />;
}

function App () {
  return (
    <div className="App">
      <Properties />
    </div>
  );
}
```